content/en/tracing/trace_collection/compatibility/go.md

```diff
@@ -32,52 +34,52 @@ Integrate the Go tracer with the following list of web frameworks using one of t
 
 **Note**: The [integrations documentation][5] provides a detailed overview of the supported packages and their APIs, along with usage examples.
 
-| Framework         | Support Type    | GoDoc Datadog Documentation                                              |
-|-------------------|-----------------|--------------------------------------------------------------------------|
-| [Gin][6]          | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gin-gonic/gin][7]               |
-| [Gorilla Mux][8] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux][9]                |
-| [gRPC][10]        | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc][11]     |
-| [gRPC v1.2][10]   | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc.v12][12] |
-| [chi][13]         | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-chi/chi][14] |
-| [echo v4][15]     | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/labstack/echo.v4][16]           |
-| [echo v3][15]     | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/labstack/echo][17]              |
-| [Fiber][18]     | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gofiber/fiber.v2][19]              |
+| Framework         | Support Type          | GoDoc Datadog Documentation                                              |
+|-------------------|-----------------------|--------------------------------------------------------------------------|
+| [Gin][6]          | Manual + Compile-Time | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gin-gonic/gin][7]               |
+| [Gorilla Mux][8]  | Manual + Compile-Time | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux][9]                 |
+| [gRPC][10]        | Manual + Compile-Time | [gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc][11]     |
+| [gRPC v1.2][10]   | Manual                | [gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc.v12][12] |
+| [chi][13]         | Manual + Compile-Time | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-chi/chi][14]                 |
+| [echo v4][15]     | Manual + Compile-Time | [gopkg.in/DataDog/dd-trace-go.v1/contrib/labstack/echo.v4][16]           |
+| [echo v3][15]     | Manual                | [gopkg.in/DataDog/dd-trace-go.v1/contrib/labstack/echo][17]              |
+| [Fiber][18]       | Manual + Compile-Time | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gofiber/fiber.v2][19]           |
```

Comment:
Should these maybe be "or" and not "+"?

Reply:
Haha `+` avoided me having to decide between `or` and `and` ... 🙃